src/azure-cli/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -129,6 +129,15 @@
         az netappfiles account list -g mygroup
 """
 
+helps['netappfiles account list-by-subscription'] = """
```

Comment:
We usually will not create ‘List-by-subscription’ command. The function should be contained in list command when user don’t add resource group  argument

Reply:
Really good point. I'll see if I can push for this change in the API but that wont be ready until next release so I think I will remove this command for this pr.

Reply:
@kairu-ms Good point. I have talked to the API team and it turns out that we were asked by a Microsoft team to implement the API and swagger this way. 
So to solve this I created a custom command which checks on the rg parameter and calls the appropriate command.